tests/check_consistent.py

```diff
@@ -46,6 +46,7 @@ def check_stdlib() -> None:
 
 def check_stubs() -> None:
     for distribution in os.listdir("stubs"):
+        assert not distribution.startswith("types-"), f"Distribution not allowed to start with 'types-': {distribution}"
```

Comment:
```suggestion
        assert not distribution.startswith("types-"), f"Directory name cannot start with 'types-': {distribution}"
```

Reply:
The directories in stubs represent distributions and the reason we disallow this is because a distribution named types-something would be confusing (for humans and for our code). So I'd prefer to keep the word "distribution" in the message.

Reply:
I'm going to merge this so I can fix the bug below, but if you still have a viewpoint on the error message I'll change it in the next PR :-)

Reply:
I feel like if anybody runs into this error, it's likely to be a new contributor, who might not understand what "distribution" means in this context. But I don't have a strong opinion :-)